src/Lykke.blue.Service.ReferralLinks.Core/Assets/IAsset.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Lykke.blue.Service.ReferralLinks.Core.Assets
+{
+    public enum Blockchain
```

Comment:
1 class = 1 file

Reply:
Moreover, this code is not used anywhere.

Reply:
Done.